src/tasks/cardid-task/cardid-task.h

```diff
@@ -0,0 +1,129 @@
+/**
+ * @file    tasks/cardid-task/cardid-task.h
+ * @brief   Card ID reading task
+ *
+ * This task uses on-board RFID reader module to try and read IDs of all cards present in the RF
+ * field. The underlying library is able to read IDs of all cards at once (if those cards behave
+ * properly).
+ *
+ * The master task must explicitly request that this task starts polling for cards. When this task
+ * is polling for cards it may read one or more card IDs in one poll cycle. When that happens,
+ * this task will invoke a callback to the Master Task and will stop polling for cards until
+ * the Master Task reenables polling.
+ *
+ * When this task is not polling for cards the RFID reader module is in low-power mode.
+ */
+
+#ifndef TASKS__CARDID_TASK_H
+#define TASKS__CARDID_TASK_H
+
+/*===========================================================================*/
+/* Task data structures and constants                                        */
+/*===========================================================================*/
+
+/**
+ * @brief       Card ID structure
+ *
+ * This structure carries the Card ID and information about how long the ID is.
+ */
+typedef struct {
+    uint8_t     uid[10];
```

Comment:
Again, defining the number as a constant would be great.

Reply:
Done